owncloud/ios-app
# [feature/whats new screen] In-App changelog "What´s new screen"

## Description
If the user has auto update enabled on the iOS device, the user cannot see the AppStore changelog, what changed in the new app version.

With this new screen the user can see an In-App changelog, when the app was started the first time, after an app update.

If there was more than one app update since the last app launch, the user will see everything what's new, since the last started app version.

It is important, that this file `ownCloud/Release Notes/ReleaseNotes.plist` needs new content before an update was submitted to Apple.
This file also contains the image data as a base64 encoded string.

The `ReleaseNotesHostViewController` can later easily used to integrate an `Intro` or `Onbording` view.

## Motivation and Context
Better marketing to the user.

## How Has This Been Tested?
- Install app version < 1.2
- Launch app
- Install app version 1.2 
- Launch app `What´s new screen` should appear
- Force quite App 
- Launch app again `What´s new screen` should NOT appear

## Screenshots (if appropriate):
![Simulator Screen Shot - iPhone 11 Pro - 2019-12-08 at 11 52 12](https://user-images.githubusercontent.com/736109/70388663-01e85600-19b5-11ea-856f-9f88a2cf970f.png)

## Types of changes


## Checklist:



## QA

Test plan:


Bugs & improvements:

- [X] (1) Showing only after upgrading https://github.com/owncloud/ios-app/pull/572#issuecomment-572457884  [DONE]
- [x] (2) Link to App Store https://github.com/owncloud/ios-app/pull/572#issuecomment-572460624 [DONE]
- [x] (3) After killing the app, what's new is there again https://github.com/owncloud/ios-app/pull/572#issuecomment-572461273 [FIXED]
